<?xml version="1.0"?>
<launch>

  <arg name="robot_name"          default="px150"/>

  <!-- pid -->
  <arg name="use_pid_cntlrs"      default="true"/>
  <arg name="gains_filepath"      default="$(find interbotix_sdk)/pid_config/$(arg robot_name)_pid.yaml"/>
  <rosparam if="$(arg use_pid_cntlrs)"
    command="load"
    file="$(arg gains_filepath)"
    ns="$(arg robot_name)"/>


    <include file="$(find interbotix_moveit)/launch/interbotix_moveit.launch">
      <arg name="robot_name"                value="$(arg robot_name)"/>
      <arg name="arm_operating_mode"        value="velocity"/>
      <arg name="arm_profile_velocity"      value="131"/>
      <arg name="arm_profile_acceleration"  value="15"/>
      <arg name="use_pid_cntlrs"            value="$(arg use_pid_cntlrs)"/>
      <arg name="use_actual"                value="true"/>
      <arg name="dof"                       value="5"/>
      <arg name="use_world_frame"           value="false"/>
      <arg name="external_urdf_loc"         value="$(find flex_gazebo_interbotix)/urdf/experiments.urdf.xacro"/>
    </include>


</launch>
